### PR TITLE
Fix missing RolloutCancelled event in notification drawer

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Fix bugzilla #1490720 ([link](https://bugzilla.redhat.com/show_bug.cgi?id=1490720))

Adds this event to the notification drawer:

screen shot 2017-10-31 at 4 05 47 pm

### CODE REVIEW

File: app/scripts/constants.js [hunk 1]
```diff
@@ -189,6 +189,7 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     BuildConfigInstantiateFailed: true,
     // Deployment
     DeploymentCancelled: true,
+    RolloutCancelled: true,
```
Comment (spadgett) [inline app/scripts/constants.js]: nit: better to keep it alphabetical by type so it's easier to find things

File: app/scripts/constants.js [hunk 2]
```diff
@@ -189,6 +189,7 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     BuildConfigInstantiateFailed: true,
     // Deployment
     DeploymentCancelled: true,
```
Comment (spadgett) [inline app/scripts/constants.js]: Does this reason exist if RolloutCancelled is for cancelled deployments

File: app/scripts/constants.js (hunk 1 above)
Comment (spadgett) [reply app/scripts/constants.js]: If this is for deployment configs, better to move it under that section

**Comment by spadgett** [comment]
I don't think `DeploymentCancelled` is a real event reason. I'd like to remove that as part of this PR.

File: app/scripts/constants.js (hunk 2 above)
Comment (benjaminapetersen) [reply app/scripts/constants.js]: My doc indicates that it did in my [first review](https://docs.google.com/document/d/11SYwEtnq0S6PT9FPXZuyktCdVsdoYfU_9lDghkTf02A/edit) of all the events, but I don't see a `c.recorder.Eventf` with `grep -ri 'DeploymentCancellationFailed' pkg` so I'm pretty sure its not real.   I'll remove this.

Comment (spadgett) [reply app/scripts/constants.js]: Might have changed from 3.6 -> 3.7

File: app/scripts/constants.js (hunk 1 above)
Comment (benjaminapetersen) [reply app/scripts/constants.js]: will do.

**Comment by benjaminapetersen** [comment]
Updated.

**Comment by spadgett** [comment]
```diff
Built /dist does not match what is committed, run 'grunt build' and include the results in your commit.
diff --git a/dist/scripts/scripts.js b/dist/scripts/scripts.js
index 354721d..2ae32d6 100644
--- a/dist/scripts/scripts.js
+++ b/dist/scripts/scripts.js
@@ -723,11 +723,10 @@ BuildCompleted: !0,
 BuildFailed: !0,
 BuildStarted: !0,
 BuildConfigInstantiateFailed: !0,
-DeploymentCancelled: !0,
-RolloutCancelled: !0,
 Failed: !0,
 DeploymentCreated: !0,
 DeploymentCreationFailed: !0,
+RolloutCancelled: !0,
 FailedRescale: !0,
 SuccessfulRescale: !0,
 BackOff: !0,
```

**Comment by benjaminapetersen** [comment]
/test

`/dist` should match.

**Comment by spadgett** [comment]
@benjaminapetersen I can look at the commit and see the dist is wrong :)

**Comment by benjaminapetersen** [comment]
:neutral_face: more :coffee: :coffee: :coffee:

**Pull request merged**